website/blog/2024-09-04-launch-week-2/index.mdx

```diff
@@ -0,0 +1,112 @@
+
+![GitOps ready](https://github.com/user-attachments/assets/491b146a-63a0-4585-93ee-aac8b7013f53)
+
+During the rest of the launch week, we will showcase a Glasskube feature that adds key components to the GitOps completeness of Glasskube.
+
+## 2\. What’s so good about GitOps anyway?
+
+Imagine if I told you you could set up a full Kubernetes setup with just one command. Glasskube bootstrap command
+
+![Glasskube GitOps command](https://github.com/user-attachments/assets/3dab678a-1ff7-4167-82b6-975070c70a0d)
+
+The Glasskube GitOps template is [officially here](https://glasskube.dev/blog/gitops-template/), think of it as a package for your packages. You can now define every single component of your cluster package stack, push it to a Git repository, and let ArgoCD handle the deployment.
+
+And it’s no exaggeration to say that deploying to Kubernetes now takes seconds instead of hours. Who wouldn’t want that?
+
+![GitOps template diagram](https://github.com/user-attachments/assets/236af5f2-64a9-42f2-92f8-a30438777365)
+
+## 3\. Hear that? …silence! But your packages are still up-to-date
+
+You might be wondering, "Wait, so I don’t have to update the packages myself?"
+
+That's right! The Glasskube package repository, the GitHub repo that stores all the packages on Glasskube Hub, runs a [Renovate](https://docs.renovatebot.com/) integration on the backend repo every day. If new versions of supported Glasskube packages are available, we test and add them to the list of versions you can choose from.
+
+![Renovate integration](https://github.com/user-attachments/assets/afc36885-6781-4cda-b8e7-7e1de59902ef)
+
+Well dive deeper into [how this works](https://glasskube.dev/docs/integration/renovate/) and discuss how you can opt in or out of the auto-updating feature.
+
+## 4\. Everybody needs a home, and so do packages
+
+Introducing [Glasskube Hub](https://glasskube.dev/products/hub/), think of it as the equivalent to Docker Hub or Artifact Hub, but for Glasskube packages. All available Glasskube packages are conveniently gathered in one place and can be consumed on demand.
+
+![Glasskube hub](https://github.com/user-attachments/assets/74dd15b1-ebdc-4071-b02e-00d802e8a71b)
+
+Apart from storing freely available Open Source Packages, the Glasskube Hub will also host private packages.
+
+Wait, what are private packages exactly?
+
+-   If your organization excels at what you do, so much so that customers are clamoring to self-host your solution, yet you lack the time, resources, or inclination to make it happen - Private Packages are the answer. We partner with you to build and host a Glasskube native package around your offering, making it incredibly simple to customize, configure, install, and update your tool within your customers infrastructure. If you’d like to learn more, [schedule a call today](https://cal.glasskube.eu/team/founder/demo?month=2024-08).
```

Comment:
```suggestion
-   If your organization excels at what you do, so much so that customers are clamoring to self-host your solution, yet you lack the time, resources, or inclination to make it happen - Private Packages are the answer. We partner with you to build and host a Glasskube native package around your offering, making it incredibly simple to customize, configure, install, and update your tool within your customers infrastructure. If you’d like to learn more, [schedule a call today](https://cal.glasskube.eu/team/founder/demo).
```

with the month query only events for august are shown.